plugins/services/src/js/utils/ServiceActionModalsUtil.ts

```diff
@@ -0,0 +1,9 @@
+import { t } from "@lingui/macro";
+
+export function getActionModalReadableError(i18n: any, errorMsg: any): any {
+  if (errorMsg === " error.min") {
+    return i18n._(t`Must be bigger than or equal to 0`);
```

Comment:
let's use trans here, so we don't have to pass the `i18n`-prop down all the way to this helper.